src/_core/chunk.ts

```diff
@@ -0,0 +1,13 @@
+export function chunk<T>(array: T[], count: number) {
```

Comment:
Is there any reason why we can't use `slice` as in this case?
https://github.com/Keeper-Wallet/Keeper-Wallet-Extension/blob/KEEP-1063/handle-data-entries-limit/src/controllers/assetInfo.ts#L352:L367

Reply:
maybe something like this?

```javascript
function* chunk(array, size) {
  for (const i = 0; i < array.length; i += size) {
    yield array.slice(i, i + size);
  }
}
```

seems to be easier to read)